src/utils/net.rs

```diff
@@ -42,7 +41,8 @@ pub async fn download_ipfs_file_trustlessly(
     {
         let mut reader = reader(url.as_str()).await?.compat();
         let mut writer = futures::io::BufWriter::new(async_fs::File::create(&tmp).await?);
-        rs_car_ipfs::single_file::read_single_file_seek(&mut reader, &mut writer, Some(cid))
+        let cid_v10 = crate::utils::cid::cid_11_to_10(cid);
+        rs_car_ipfs::single_file::read_single_file_seek(&mut reader, &mut writer, Some(&cid_v10))
```

Comment:
`cid_0_10` is needed until https://github.com/dapplion/rs-car/pull/13 is merged and released.